site/content/blogs/release-v118.en.md

```diff
@@ -54,6 +192,20 @@ Under the hood, Copilot sets [minimumHealthyPercent and maximumPercent](https://
 ## Tracing for Load Balanced Web Service, Worker Service, and Backend Service
 _Contributed by [Danny Randall](https://github.com/dannyrandall/)_
 
+In [v1.17](./release-v117.en.md#send-your-request-driven-web-services-traces-to-aws-x-ray), Copilot launched support for sending traces from your Request-Driven Web Services to [AWS X-Ray](https://aws.amazon.com/xray/).
+Now, you can easily export traces from your Load Balanced Web, Worker, and Backend services to X-Ray by modifying your service's manifest:
+```yaml
+observability:
+  tracing: awsxray
+```
+
+For these services, Copilot will deploy an [AWS Distro for OpenTelemetry Collector](https://github.com/aws-observability/aws-otel-collector) sidecar container to collect traces from your service and export them to X-Ray.
```

Comment:
```suggestion
For these services, Copilot will deploy an [AWS Distro for OpenTelemetry Collector](https://github.com/aws-observability/aws-otel-collector) sidecar container to collect traces from your services and export them to X-Ray.
```

Reply:
how about this? 
```suggestion
For these service types, Copilot will deploy an [AWS Distro for OpenTelemetry Collector](https://github.com/aws-observability/aws-otel-collector) sidecar container to collect traces from your service and export them to X-Ray.
```

i think leaving it singular makes more sense because a collector is deployed per service - what do you think?